### PR TITLE
DROOLS-2102: Use Child objects in scenario to fire rule

### PR DESCRIPTION
**Pull request by jomarko**
@Rikkola @manstis Please have a look.

### CODE REVIEW

File: drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-backend/src/test/java/org/drools/workbench/screens/testscenario/backend/server/model/Acid.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+package org.drools.workbench.screens.testscenario.backend.server.model;
+
+
+public class Acid implements java.io.Serializable {
+
+	static final long serialVersionUID = 1L;
+
+	private java.lang.String name;
+	private java.util.List<org.drools.workbench.screens.testscenario.backend.server.model.Molecule> molecules;
```
Comment (manstis) [inline drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-backend/src/test/java/org/drools/workbench/screens/testscenario/backend/server/model/Acid.java]: An ```import``` would have been nice.. but not essential.

Comment (manstis) [reply drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-backend/src/test/java/org/drools/workbench/screens/testscenario/backend/server/model/Acid.java]: (for both ```List``` and ```Molecule``` and later ```String```)

**Comment by jomarko** [comment]
@manstis commit added, shout if anything else I could improve.

File: drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-backend/src/test/java/org/drools/workbench/screens/testscenario/backend/server/model/Acid.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.drools.workbench.screens.testscenario.backend.server.model;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class Acid implements Serializable {
```
Comment (manstis) [inline drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-backend/src/test/java/org/drools/workbench/screens/testscenario/backend/server/model/Acid.java]: Would you mind revisiting the formatting?

File: drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-backend/src/test/java/org/drools/workbench/screens/testscenario/backend/server/model/Acid.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public List<Molecule> getMolecules() {
+        return this.molecules;
+    }
+
+    public void setMolecules(
```
Comment (manstis) [inline drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-backend/src/test/java/org/drools/workbench/screens/testscenario/backend/server/model/Acid.java]: Formatting here...

File: drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-backend/src/test/java/org/drools/workbench/screens/testscenario/backend/server/model/Acid.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            List<Molecule> molecules) {
+        this.molecules = molecules;
+    }
+
+    public Acid(
```
Comment (manstis) [inline drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-backend/src/test/java/org/drools/workbench/screens/testscenario/backend/server/model/Acid.java]: Formatting here...

File: drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-backend/src/test/java/org/drools/workbench/screens/testscenario/backend/server/model/Acid.java (hunk 2 above)
Comment (jomarko) [reply drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-backend/src/test/java/org/drools/workbench/screens/testscenario/backend/server/model/Acid.java]: Should be done now, I copied the java model classes from workbench data editor. It is where comes these weird things from. Also weird after that I applied code style in IDE and not all of them were corrected.

**Pull request merged**